src/content/pages/programme/voting.mdx

```diff
@@ -17,13 +17,13 @@ ingredient in this refinement and curation process.
 
 <div style={{textAlign: "center"}}>
 
-Community Voting is open
+Community Voting is <b>closed</b>.
 
 from Wednesday, 5 February 23:00:00 UTC
 
 until Monday, 17 February 23:59:59 UTC
 
-<ButtonLink url="https://voting.europython.eu">Cast your votes now!</ButtonLink>
+<b>Thanks for your votes!</b>
```

Comment:
We have a disabled prop for the ButtonLink now, if you'd like to give it a try 🙂

```suggestion
<ButtonLink disabled>Thanks for your votes!</ButtonLink>
```